<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureData</class>
 <widget class="QDialog" name="FeatureData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>feature data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_dialogName">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;observations, readings and statistic of the selected feature&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_selectedFeature">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_sensorInformation">
      <attribute name="title">
       <string>sensor information</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_activeSensor">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>configuration</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_Configname">
               <property name="text">
                <string>config name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_configName"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_accuracy">
          <attribute name="title">
           <string>accuracy</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QToolBox" name="toolBox_accuracy">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_polar">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>106</width>
                <height>105</height>
               </rect>
              </property>
              <attribute name="label">
               <string>polar</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaAzimuth">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaAzimuth"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaZenith">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaZenith"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaDistance">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaDistance"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_cartesian">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>106</width>
                <height>105</height>
               </rect>
              </property>
              <attribute name="label">
               <string>cartesian</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaX">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaX"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaY">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaY"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaZ">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaZ"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_temperature">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>106</width>
                <height>43</height>
               </rect>
              </property>
              <attribute name="label">
               <string>temperature</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaTemperature">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaTemperature"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_levelMeasurement">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>106</width>
                <height>74</height>
               </rect>
              </property>
              <attribute name="label">
               <string>level measurement</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaXZ">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaXZ"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1">
                 <item>
                  <widget class="QLabel" name="label_sigmaYZ">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_sigmaYZ"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_sigmaUndefined">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>98</width>
                <height>28</height>
               </rect>
              </property>
              <attribute name="label">
               <string>others</string>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_sensorConfiguration">
          <attribute name="title">
           <string>sensor configuration</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_connection">
          <attribute name="title">
           <string>connection</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_connectionType">
               <property name="text">
                <string>connection type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_connectiontype"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_ip">
               <property name="text">
                <string>ip</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_ip"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_port">
               <property name="text">
                <string>port</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_port"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_comport">
               <property name="text">
                <string>comport</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_comport"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_baudrate">
               <property name="text">
                <string>baudrate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_baudrate"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_databits">
               <property name="text">
                <string>databits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_databits"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_flowcontrol">
               <property name="text">
                <string>flowcontrol</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_flowcontrol"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_parity">
               <property name="text">
                <string>parity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_parity"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_stopbits">
               <property name="text">
                <string>stopbits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_stopbits"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_connectionChanged">
             <property name="text">
              <string>connection changed</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_setSensorConfig">
         <property name="text">
          <string>set sensor configuration</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="observations">
      <attribute name="title">
       <string>observations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableView_observation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="readings">
      <attribute name="title">
       <string>readings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableView_readings"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_statistic">
      <attribute name="title">
       <string>statistic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_displayFunction">
           <property name="text">
            <string>displayed function statistic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_displayedFunction"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_displayedfunctionStatistic"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_covarianceMatrices">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_sigmaXXaposteriori">
          <attribute name="title">
           <string>Qxx á posteriori</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_s0aposteriori">
               <property name="text">
                <string>s0 á posteriori:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_s0aposterioriValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_qxxAposteriori"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_sigmaXXapriori">
          <attribute name="title">
           <string>Sxx á priori</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_s0apriori">
               <property name="text">
                <string>s0 á priori</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_s0aprioriValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_sxxApriori"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_protocol">
      <attribute name="title">
       <string>protocol</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextBrowser" name="textBrowser_protocol"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
